src/firefly/js/visualize/ui/StretchDropDownView.jsx

```diff
@@ -90,10 +90,12 @@ function stretchByType(pv,currRV,sType,min,max) {
 }
 
 
-
+/*
+ *  LZ 9/21/16  DM-7695
+ */
 export function StretchDropDownView({plotView:pv}) {
     var enabled= pv ? true : false;
-    var rv= primePlot(pv).plotState.getRangeValues();
+    var rv= primePlot(pv).plotState.getPrimaryRangeValues();//getRangeValues();//LZ 9/21/16
```

Comment:
I changed the line back as what it was:  "var rv= primePlot(pv).plotState.getRangeValues()".  But the color stretch box can no longer be opened because PlotState.js has not getRangeValues method. When "getPrimaryRangeValues()" is used, the color stretch box can be opened.